components/terrain/world.hpp

```diff
@@ -40,36 +40,48 @@ namespace Terrain
     class ChunkManager;
     class CompositeMapRenderer;
 
-class HeightCullCallback : public osg::NodeCallback
-{
-public:
-    HeightCullCallback() : mLowZ(-std::numeric_limits<float>::max()), mHighZ(std::numeric_limits<float>::max()), mMask(~0) {}
-
-    void setLowZ(float z)
+    class HeightCullCallback : public osg::NodeCallback
     {
-        mLowZ = z;
-    }
-    float getLowZ() const { return mLowZ; }
-
-    void setHighZ(float highZ)
-    {
-        mHighZ = highZ;
-    }
-    float getHighZ() const { return mHighZ; }
-
-    void setCullMask(unsigned int mask) { mMask = mask; }
-    unsigned int getCullMask() const { return mMask; }
-
-    virtual void operator()(osg::Node* node, osg::NodeVisitor* nv)
-    {
-        if (mLowZ <= mHighZ)
-            traverse(node, nv);
-    }
-private:
-    float mLowZ;
-    float mHighZ;
-    unsigned int mMask;
-};
+    public:
+        HeightCullCallback() : osg::NodeCallback() {}
```

Comment:
```suggestion
```
will do the same.

Reply:
Never quite understood constructor inheritance rules.